pavelib/tests.py

```diff
@@ -70,6 +70,19 @@
         dest='disable_migrations',
         help="Create tables by applying migrations."
     ),
+    make_option(
+        '--disable_courseenrollment_history',
+        action='store_true',
+        dest='disable_courseenrollment_history',
+        help="Disable history on student.CourseEnrollent. Can also be used by exporting"
+             "DISABLE_COURSEENROLLMENT_HISTORY=1."
+    ),
+    make_option(
+        '--enable_courseenrollment_history',
+        action='store_false',
+        dest='disable_courseenrollment_history',
+        help="Enable django-simple-history on student.CourseEnrollment."
+    ),
```

Comment:
Why are both of these options needed here and below? Shouldn't the history default to enabled and only require the `--disable_courseenrollment_history` to disable history?

Reply:
I was following the pattern set by `--disable_migrations` above. I suppose it's to allow explicit setting either way, and doesn't seem to harm anything.